models/engine/file_storage.py

```diff
@@ -68,3 +68,29 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Retrieve one object
+
+        Args:
+          cls: string representing the class name
+          id: string representing the object ID
+        Returns:
+          The object based on the class name and its ID, or None if not found
+        """
+        objects = self.all().values()
+        for obj in objects:
+            if id == obj.id:
+                return obj
+
+    def count(self, cls=None):
+        """Count the number of objects in storage
+
+        Args:
+          cls: string representing the class name (optional)
+        Returns:
+          The number of objects in storage matching the given class name
+          If no name is passed, returns the count of all objects in storage
+        """
+        obj_dict = self.all(cls)
+        return len(obj_dict)
```

Comment:
Good job with making everything modular! I like how I can use file_storage or db_storage without needing to change much on the user's side.